apps/board/src/app.module.ts

```diff
@@ -25,7 +27,11 @@ import mysqlConfig from './config/mysql.config';
           logging: true,
           synchronize: process.env.NODE_ENV === 'development',
         };
-        return boardTypeOrmModuleOptions;
+      },
+      dataSourceFactory: async (options: DataSourceOptions): Promise<DataSource> => {
+        if (!options) throw new Error('Invalid options passed');
+        const dataSource = new DataSource(options);
+        return addTransactionalDataSource(dataSource);
```

Comment:
슬랙에서 비슷한 얘기를 나눴었는데 한번 확인해 주셔도 좋을 것 같습니다~
결론적으로는 options parameter는 optional로 표현되는게 좋다는 생각입니다!
https://greppkdtreactnodejs.slack.com/archives/C07JDB0K0VD/p1727601893856479
```suggestion
      dataSourceFactory: async (options?: DataSourceOptions): Promise<DataSource> => {
        if (!options) throw new Error('Invalid options passed');
        const dataSource = new DataSource(options);
        return addTransactionalDataSource(dataSource);
```

Reply:
감사합니다! 
해당 부분 반영했습니다 : 01dbb08178ca3a023d60a5b7528406f3fed6a72e